Moblin/View/Settings/RtmpServer/RtmpServerStreamSettingsView.swift

```diff
@@ -136,12 +133,13 @@ struct RtmpServerStreamSettingsView: View {
                 }))
                 .disabled(model.rtmpServerEnabled())
                 if stream.manualFps! {
-                    TextEditNavigationView(
-                        title: String(localized: "FPS"),
-                        value: String(stream.fps!),
-                        onSubmit: submitFps,
-                        keyboardType: .numbersAndPunctuation
-                    )
+                    Picker(String(localized: "FPS"), selection: Binding(get: {
+                        String(stream.fps!)
+                    }, set: submitFps)) {
+                        ForEach(netStreamFps, id: \.self) {
+                            Text($0)
+                        }
+                    }
```

Comment:
Floating point numbers are dangerous. You sure no weird rounding errors will occur?